components/esm/aipackage.cpp

```diff
@@ -7,8 +7,7 @@ namespace ESM
 {
     void AIData::blank()
     {
-        mHello = mU1 = mFight = mFlee = mAlarm = mU2 = mU3 = mU4 = 0;
-        mServices = 0;
+        *this = AIData();
```

Comment:
It works correctly. I have tested that. There is no value-initializaion. AIData - POD structure, when you create a variable of POD type compiler does not initialize it for you. `*this = AIData();` equal to calling constructor and applying to itself. All fields will be zeros